src/julia-parser.scm

```diff
@@ -1027,7 +1027,7 @@
                                     (parse-string-literal s #t)))
                         (nxt (peek-token s))
                         (macname (symbol (string #\@ ex '_str)))
-                        (macstr (if (triplequote-string-literal? str) str (cadr str))))
```

Comment:
Over terseness can be an impediment to communication.  In-line comments explaining the structure that you are processing might have made the change clear (not just for me, but for anybody else who needs to maintain this in the future).